### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
### Checklist - did you ...

- [X] ~Add a file to the `news` directory ([using the template](https://github.com/conda/conda/blob/master/news/TEMPLATE)) for the next release's release notes?~
     
- [X] ~Add / update necessary tests?~
- [X] ~Add / update outdated documentation?~

### CODE REVIEW

**Comment by beeankha** [comment]
Hi @kianmeng , please update via a comment on here once you sign the Contributor License Agreement.

**Comment by kianmeng** [comment]
@beeankha Signed, please check again.

**Comment by kenodegard** [comment]
@anaconda-issue-bot check

File: conda/_vendor/toolz/itertoolz.py [hunk 1]
```diff
@@ -728,7 +728,7 @@ def partition_all(n, seq):
 def count(seq):
     """ Count the number of items in seq
 
-    Like the builtin ``len`` but works on lazy sequencies.
+    Like the builtin ``len`` but works on lazy sequences.
```
Comment (kenodegard) [inline conda/_vendor/toolz/itertoolz.py:731]: Please do not modify vendored code.

Comment (kianmeng) [reply conda/_vendor/toolz/itertoolz.py]: @kenodegard Reverted.

**Review by kenodegard — APPROVED** [review]

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**